### PR TITLE
Add link to e10s explanation page

### LINKED ISSUE #34: Add explanation what "multi-process" is
@brianking suggested we add a link to the panel explaining what e10s or _multi-process_ is. I currently do not know of such a page, maybe @atsay knows if there is one.

If not, it might just be a matter of taking some content from

* https://developer.mozilla.org/en-US/Firefox/Multiprocess_Firefox (intro text)
* https://developer.mozilla.org/en-US/Firefox/Multiprocess_Firefox/Motivation
* https://billmccloskey.wordpress.com/2013/12/05/multiprocess-firefox/

Who could help writing that page?

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #34

### CODE REVIEW

File: data/reporter.js [hunk 1]
```diff
@@ -53,7 +53,13 @@ var initialize = function(data) {
 
         var e10s = document.createElement("div");
         e10s.setAttribute("class", "version");
-        e10s.appendChild(document.createTextNode((document.addonReports[i].multiprocessCompatible ? "Compatible" : "Not compatible") + " with multi-process."));
+        var e10s_link = document.createElement("a");
+        e10s_link.setAttribute("href", "#");
+        e10s_link.addEventListener("click", function() { self.port.emit("openE10sIntroLink"); });
```
Comment (tofumatt) [inline data/reporter.js]: Can you put the contents of the function on the next line? It makes it easier to read.

```js
e10s_link.addEventListener("click", function() {
    self.port.emit("openE10sIntroLink");
});
```

File: data/reporter.js [hunk 2]
```diff
@@ -98,7 +104,13 @@ var initialize = function(data) {
 };
 
 self.port.on("app_e10s_enabled", function(enabled) {
-    $("#appE10sEnabled").text("Multi-process is " + (enabled ? "" : "not ") + "enabled.");
+    $("#appE10sEnabled").empty();
+    var e10s_link = document.createElement("a");
+    e10s_link.setAttribute("href", "#");
+    e10s_link.addEventListener("click", function() { self.port.emit("openE10sIntroLink"); });
```
Comment (tofumatt) [inline data/reporter.js]: Same here re: one-line-functions.

File: data/reporter.js [hunk 3]
```diff
@@ -53,7 +53,13 @@ var initialize = function(data) {
 
         var e10s = document.createElement("div");
         e10s.setAttribute("class", "version");
-        e10s.appendChild(document.createTextNode((document.addonReports[i].multiprocessCompatible ? "Compatible" : "Not compatible") + " with multi-process."));
+        var e10s_link = document.createElement("a");
+        e10s_link.setAttribute("href", "#");
```
Comment (tofumatt) [inline data/reporter.js]: Can this not just be a link? Or will it open in the ACR panel that way?

**Comment by tofumatt** [comment]
r+wc

File: data/reporter.js (hunk 3 above)
Comment (wagnerand) [inline data/reporter.js]: Exactly. If I use `target='_blank'` it will open a new window, ugh.

**Comment by wagnerand** [comment]
Thank you!

**Pull request merged**